opal/mca/hwloc/Makefile.am

```diff
@@ -10,7 +10,7 @@
 #
 
 # We do not want -I$(srcdir) in AM_CPPFLAGS, or there can be a
-# conflict between system hwloc.h and opal/mca/hwloc/hwloc.h.  So just
+# conflict between system hwloc.h and opal/mca/hwloc/hwloc-internal.h.  So just
```

Comment:
You're exactly right -- we don't.  I just noticed this when one of the CI bots failed "make dist".  I have a fix (where we don't set `AM_CPPFLAGS` at all), but I'm doing a little more testing before I push it up here.